github/workflow.go

```diff
@@ -78,3 +75,18 @@ func GetWorkflowContext() (WorkflowContext, error) {
 	err := json.Unmarshal([]byte(ghContext), &w)
 	return w, err
 }
+
+// GetToken gets the Github Actions token.
+// See: https://docs.github.com/en/actions/security-guides/automatic-token-authentication
+func GetToken() (string, error) {
+	var w struct {
+		Token string `json:"token,omitempty"`
+	}
+	ghContext, ok := os.LookupEnv(githubContextEnvKey)
```

Comment:
I think you can simply use `os.LookupEnv("GITHUB_TOKEN")`

Reply:
My understanding is that you still need to add at it to you env like `GITHUB_TOKEN: {{ secrets.GITHUB_TOKEN }}` or `GITHUB_TOKEN: {{ github.token }}` but we're already adding the full `github` context anyway.

I don't see `GITHUB_TOKEN` in the default env vars: https://docs.github.com/en/actions/learn-github-actions/environment-variables

Reply:
It should be available in the re-usable workflow. Interesting that it's not listed in the default env variables, though. SO maybe you're right.

Reply:
Yeah, I've seen some places online that refer to it without explicitly adding it to env so 🤷 

In any case getting it from the `github` context works.